OSVR-Unity/CHANGES.md

```diff
@@ -4,6 +4,18 @@ This is an abbreviated changelog for the OSVR Unity Plugin.
 
 Use git for a full changelog.
 ##Recent Changes
+### ClientKit Rendering Parameters update
+> 13-September-2015 (commit 34fd6f1) v0.2.119-g34fd6f1
+
+- This update simplifies the Unity plugin by retrieving the output of the computational display model (viewport, projection matrices) from the OSVR-Core API. This eliminates the need to parse JSON display descriptor data in Unity, which allows for improvements in the display model without having to rebuild a game. 
+
+- VRHead.cs has been renamed VRViewer.cs to fit with the display model. Conceptually, a VRViewer has one or more VREyes, and each VREye has a VRSurface which controls rendering (has the camera component) for that VREye. In the scene hierarchy, VRViewer and VREyes are siblings, although conceptually this is a parent-child relationship. The reason for this is because VREye poses are reported in world space, not head space. VRSurfaces are children of VREyes, both conceptually and in the scene hierarchy.
```

Comment:
Perhaps note that devs will never need to create any of these objects manually or see them in their project, with a "see below" note?